### PR TITLE
update 21.12 branch for doc [skip ci]

### PR DESCRIPTION
**Pull request by viadea**
update 21.12branch for doc
Signed-off-by: Hao Zhu <hazhu@nvidia.com>

### CODE REVIEW

**Review by jlowe — DISMISSED** [review]

File: docs/get-started/getting-started-on-prem.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ CUDA and will not run on other versions. The jars use a maven classifier to keep
 
 For example, here is a sample version of the jars and cudf with CUDA 11.0 support:
 - cudf-21.12.0-cuda11.jar
-- rapids-4-spark_2.12-21.12.0-SNAPSHOT.jar
+- rapids-4-spark_2.12-21.12.0.jar
```
Comment (jlowe) [inline docs/get-started/getting-started-on-prem.md:59]: Other docs aren't removing SNAPSHOT but this one is

Comment (viadea) [reply docs/get-started/getting-started-on-prem.md:59]: Hi Jason, this is used by the example I think we should change it right?

Comment (jlowe) [reply docs/get-started/getting-started-on-prem.md:59]: The other occurrences where you removed `SNAPSHOT` are also examples, so what makes this example different than the others?  Note that when one builds on branch-21.12, they will get a SNAPSHOT jar and arguably we should use SNAPSHOT here.  Docs on the `main` branch would not, and there's separate commits, only to `main`, that remove `SNAPSHOT` from both docs and the pom versions.

Comment (viadea) [reply docs/get-started/getting-started-on-prem.md]: Sure @jlowe  i just added the snapshot back.

**Review by jlowe — APPROVED** [review]

**Review by sameerz — APPROVED** [review]
In a future release we ought to consider updating the docs/demo/Databricks/generate-init-script-cuda11.ipynb to use a more recent toolkit and driver (at least driver 450.80.02 or higher).

**Pull request merged**